### PR TITLE
Renamed tests for cli/test_template.py.

### PR DESCRIPTION
**Pull request by omaciel**
Test case naming follows #3038 conventions.

### CODE REVIEW

**Comment by omaciel** [comment]
```shell
(robottelo)┌[omaciel☮Ogs-MacBook-Pro.local]-(~/hacking/robottelo)-[git://rename-tests-cli-template ✔]-
└> py.test -v tests/foreman/cli/test_template.py --collect-only
================================================================= test session starts ==================================================================
platform darwin -- Python 2.7.10, pytest-2.8.3, py-1.4.31, pluggy-0.3.1 -- /Users/omaciel/.virtualenvs/robottelo/bin/python
cachedir: .cache
rootdir: /Users/omaciel/hacking/robottelo, inifile:
collected 9 items
<Module 'tests/foreman/cli/test_template.py'>
  <UnitTestCase 'TemplateTestCase'>
    <TestCaseFunction 'test_positive_add_os_by_id'>
    <TestCaseFunction 'test_positive_create_locked'>
    <TestCaseFunction 'test_positive_create_with_content'>
    <TestCaseFunction 'test_positive_create_with_loc'>
    <TestCaseFunction 'test_positive_create_with_name'>
    <TestCaseFunction 'test_positive_create_with_org'>
    <TestCaseFunction 'test_positive_delete_by_id'>
    <TestCaseFunction 'test_positive_remove_os_by_id'>
    <TestCaseFunction 'test_positive_update_name'>

===================================================================  in 0.33 seconds ===================================================================
```

**Comment by sghai** [comment]
ACK.. Looks good.

File: tests/foreman/cli/test_template.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def test_create_template_with_org(self):
 
     @run_only_on('sat')
     @tier2
-    def test_add_operating_system_1(self):
+    def test_positive_add_os_by_id(self):
         """@Test: Check if Template can be assigned operating system
```
Comment (oshtaier) [inline tests/foreman/cli/test_template.py]: I know it is out of scope, but I can't pass by:
`Check if operating system can be assigned to template by id`

File: tests/foreman/cli/test_template.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def test_add_operating_system_1(self):
 
     @run_only_on('sat')
     @tier2
-    def test_remove_operating_system_1(self):
+    def test_positive_remove_os_by_id(self):
         """@Test: Check if OS can be removed Template
```
Comment (oshtaier) [inline tests/foreman/cli/test_template.py]: I know it is out of scope, but I can't pass by:
`Check if operating system can be removed from template by id`

**Comment by oshtaier** [comment]
ACK pending non-blocking comments

**Comment by omaciel** [comment]
@oshtaier agreed. Updated.

**Pull request merged**